tools/webpack-jumpstart

```diff
@@ -107,10 +107,16 @@ if [ "$(git rev-parse HEAD^{tree})" != "${target_tree}" ]; then
     fi
 fi
 
+# At this point, the only way that the working tree is different from
+# ${target_tree} is if --best was used.
+
 tools/node-modules make_package_lock_json
 unpack_from_cache "${tag}"
 
 for file in $(git diff --name-only "${target_tree}"); do
-    message TOUCH "${file}"
-    touch "${file}"
+    for match in $(grep -slP "^\t${file} \\\\$" dist/*/Makefile.deps); do
```

Comment:
This is some serious duplication of `make` rules and logic. I'd rather not want this to happen without `--best`. Can you please conditionalize the whole paragraph on `[ -n `${best} ]`?

Reply:
I got rid of this approach: we just touch all the files now.